src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -188,7 +188,9 @@ public static SvnInfoResult GetSvnFileInfo(this ICakeContext context, FilePath f
         /// <param name="context">The Cake context.</param>
         /// <param name="filePath">The file.</param>
         /// <param name="settings">The settings.</param>
-        /// <returns>The Subversion information.</returns>
+        /// <returns>The Subversion information. If no information about the file is available on Subversion 
+        /// <c>null</c> will be returned. For this <see cref="SvnSettings.ThrowOnError"/> has to be set to <c>false</c> 
+        /// and pass as parameter to <see cref="GetSvnFileInfo(ICakeContext, FilePath, SvnInfoSettings)"/>.</returns>
```

Comment:
Remove last part (and pass as parameter...), which doesn't add any value since this already is the documentation for this method.